x-pack/plugins/search_assistant/public/components/app.tsx

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import { KibanaPageTemplate } from '@kbn/shared-ux-page-kibana-template';
+export const App: React.FC = () => {
+  return (
+    <KibanaPageTemplate.Section
+      alignment="top"
+      restrictWidth={false}
+      grow
+      css={{
```

Comment:
We should remove the custom CSS and the contentProps, as well as the custom class name. Keep this as standard and simple as possible.